### PR TITLE
BZ 1681175: kubelet crash known issue

### PR DESCRIPTION
**Pull request by ousleyp**
https://bugzilla.redhat.com/show_bug.cgi?id=1681175

@openshift/team-documentation PTAL 💖

### CODE REVIEW

File: cnv_release_notes/cnv_release_notes.adoc [hunk 1]
```diff
@@ -123,6 +123,22 @@ $ virtctl image-upload ... image.raw.gz
...
 https://bugzilla.redhat.com/show_bug.cgi?id=1673921[(BZ#1673921)]
 
+* If the kubelet on a node crashes or restarts, this causes the kubelet to
+incorrectly report 0 KVM devices. Virtual machines are not properly scheduled
+on affected nodes.
++
+Verify the number of devices that the kubelet reports by running
+`$ oc get node $NODE | grep devices.kubevirt`. The output on an affected node
+shows `devices.kubevirt.io/kvm: 0`.
+https://bugzilla.redhat.com/show_bug.cgi?id=1681175[(BZ#1681175)]
```
Comment (ahardin-rh) [inline cnv_release_notes/cnv_release_notes.adoc]: We tend to format BZ #s for bugs and known issues as:

    (link:https://bugzilla.redhat.com/show_bug.cgi?id=1681175[*BZ#1681175*])

File: cnv_release_notes/cnv_release_notes.adoc [hunk 2]
```diff
@@ -123,6 +123,22 @@ $ virtctl image-upload ... image.raw.gz
 Restarting a virtual machine while it is in the *Importing* state results in an error.
 https://bugzilla.redhat.com/show_bug.cgi?id=1673921[(BZ#1673921)]
 
+* If the kubelet on a node crashes or restarts, this causes the kubelet to
+incorrectly report 0 KVM devices. Virtual machines are not properly scheduled
+on affected nodes.
++
+Verify the number of devices that the kubelet reports by running
+`$ oc get node $NODE | grep devices.kubevirt`. The output on an affected node
```
Comment (ahardin-rh) [inline cnv_release_notes/cnv_release_notes.adoc]: by running:
    +
    ----
    $ oc get node $NODE | grep devices.kubevirt
    ----
    +
    The output . . . 

If you don't like that format, then you can keep the command as is, but drop the $

**Comment by ahardin-rh** [comment]
@ousleyp Just some minor suggestions. :+1:

File: cnv_release_notes/cnv_release_notes.adoc (hunk 1 above)
Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: Thanks for the heads up! I changed the link formatting throughout the Release Notes.

File: cnv_release_notes/cnv_release_notes.adoc (hunk 2 above)
Comment (ousleyp) [reply cnv_release_notes/cnv_release_notes.adoc]: Makes sense! Thanks for the suggestion. Fixed in https://github.com/openshift/openshift-docs/pull/13983/commits/368d71865fde6820944fb25ef98a83f6d9f4de9a

**Pull request merged**